daemon/containerd/image.go

```diff
@@ -157,17 +157,26 @@ func (i *ImageService) GetImageManifest(ctx context.Context, refOrID string, opt
 
 	cs := i.client.ContentStore()
 
-	desc, err := i.resolveDescriptor(ctx, refOrID)
+	image, err := i.resolveImage(ctx, refOrID)
```

Comment:
This variable collides with the `image` import

Reply:
Changed it thanks, yeah it would be great if we were able to enable this lint